webui/react/package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "scripts": {
     "postshrinkwrap": "npx npm-force-resolutions && npm install --ignore-scripts",
-    "start": "react-app-rewired start",
+    "start": "ESLINT_NO_DEV_ERRORS=true react-app-rewired start",
```

Comment:
could we set this in makefile in a way that is overridable?

Reply:
probably having it the same way as how `PUBLIC_URL` is set might also let us not need the `.env` file

Reply:
I was split on that versus baking it in. Chose the latter just because I couldn't think of a case where I would want my dev environment to not run if there are errors, but can't hurt to have it as an option to keep this area consistent.

Reply:
> I would want my dev environment to not run if there are errors,

you're right I don't see a use for that either it's just that setting it the old way does not respect the user's wish to set that var. minor difference anyway :)